backend/config.ts

```diff
@@ -38,6 +38,7 @@ export const app = {
     enabledPrisons: process.env.HOMEPAGE_REDIRECT_ENABLED_PRISONS || '',
     scheduleRedirectForPrisons: process.env.HOMEPAGE_SCHEDULE_REDIRECT_FOR_PRISONS || '',
   },
+  gitRef: process.env.GIT_REF || '',
```

Comment:
should really be `requiredInProduction` - see https://github.com/ministryofjustice/hmpps-template-typescript/blob/main/server/config.ts#L35

Reply:
Yes, DPS does not have a lot of things that the template does